pages/changelog.js

```diff
@@ -0,0 +1,107 @@
+import { CalendarIcon } from "@heroicons/react/20/solid";
```

Comment:
The file already exists, you only need to add a new object entry here 
https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/changelog.js#L12